pygmt/tests/test_plot.py

```diff
@@ -93,30 +93,19 @@ def test_plot_fail_no_data(data):
         )
 
 
-def test_plot_fail_size_color(data):
+def test_plot_fail_color_size_intensity(data):
```

Comment:
I rename this test function (also see https://github.com/GenericMappingTools/pygmt/pull/1065#discussion_r596161372) Do you think it okay?